src/less/game-end/index.less

```diff
@@ -0,0 +1,53 @@
+.gameend {
+  z-index: 100001;
+  position: absolute;
+  margin: auto;
+  width: 50%;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  height: 50%;
```

Comment:
what about a min-height, so we are sure all components will be actually on the layer